### PR TITLE
Handle EventHandler extra kwagrs arguments.

### PR DESCRIPTION
**Pull request by akash-vns**
represented PR for - https://github.com/django-oscar/django-oscar/issues/3749
 
 Here we can accept the extra kwargs with ```get_handler_kwargs()``` method.

### CODE REVIEW

File: src/oscar/apps/order/mixins.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+    def get_handler(self, user=None, **kwargs):
+        """Return instance of handler"""
+        handler_kwargs = self.get_handler_kwargs()
+        handler_kwargs.update(**kwargs)
+        return self.get_handler_class()(user, **handler_kwargs)
+
+    def get_handler_kwargs(self):
+        """Return kwargs arguments for instantiating handler"""
+        kwargs = dict()
```
Comment (solarissmoke) [inline src/oscar/apps/order/mixins.py:25]: Please change this to `return {}`

**Review by solarissmoke — CHANGES_REQUESTED** [review]

File: src/oscar/apps/order/mixins.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
+from oscar.core.loading import get_class
+
+EventHandler = get_class('order.processing', 'EventHandler')
+
+
+class EventHandlerMixin(object):
```
[the remark this answers is not in the document]
Comment (kaushaltrootech) [reply src/oscar/apps/order/mixins.py:6]: Okay

File: src/oscar/apps/order/mixins.py [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    """
```
[the remark this answers is not in the document]
Comment (kaushaltrootech) [reply src/oscar/apps/order/mixins.py:7]: Okay sure

File: src/oscar/apps/order/mixins.py (hunk 1 above)
Comment (kaushaltrootech) [reply src/oscar/apps/order/mixins.py:25]: Okay sure

File: src/oscar/apps/order/mixins.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (solarissmoke) [reply src/oscar/apps/order/mixins.py]: This doesn't seem to have been addressed?

**Pull request merged**